wayrs-client/src/connection.rs

```diff
@@ -37,14 +36,16 @@ pub enum ConnectError {
 /// and dispatches object events.
 ///
 /// Set `WAYLAND_DEBUG=1` environment variable to get debug messages.
-pub struct Connection<D> {
+pub struct Connection<D, T: ClientTransport = UnixStream> {
     #[cfg(feature = "tokio")]
     async_fd: Option<AsyncFd<RawFd>>,
+    #[cfg(feature = "tokio")]
+    read_async_fd: Option<AsyncFd<RawFd>>,
```

Comment:
If this is because of the issue you mentioned in https://github.com/MaxVerevkin/wayrs/issues/8#issuecomment-1998653089, this should not be necessary anymore.